templates/opencraft-nginx.j2

```diff
@@ -0,0 +1,57 @@
+server {
+    listen *:80;
+
+    server_name {{ OPENCRAFT_DOMAIN_NAME }} www.{{ OPENCRAFT_DOMAIN_NAME }};
```

Comment:
I lost some time due to this. Nginx is configured in such a way that first server block it encounters is treated as the default one, which responds to requests for domains not matched by any other servers. So this server responds to every host, instead of only: `{{ OPENCRAFT_DOMAIN_NAME }}` and `www.{{ OPENCRAFT_DOMAIN_NAME }}`. 

Solution is to include default server responding with 404 for every request, like on this [SO question](http://stackoverflow.com/questions/9824328/why-is-nginx-responding-to-any-domain-name).